ios/Scripts/install_fitchef_animation.sh

```
@@ -0,0 +1,173 @@
+set -euo pipefail
+
+# Скрипт для установки анимации FitChef (4 кадра)
+
+# Обработка аргументов командной строки
+FORCE_MODE="false"
+NON_INTERACTIVE="false"
+
+while [[ $# -gt 0 ]]; do
+    case $1 in
+        --force)
+            FORCE_MODE="true"
```

Shellcheck (0.11.0)

[warning] FORCE_MODE appears unused. Verify use (or export if used externally).

(SC2034)